src/main/java/net/revelc/code/formatter/FormatterMojo.java

```diff
@@ -1074,6 +1098,9 @@ static class ResultCollector {
         /** The skipped count. */
         int skippedCount;
```

Comment:
```suggestion
        /** The skipped count is incremented for cached files that haven't changed since being cached. */
        int skippedCount;
```